.ci/ansible/tasks/install_deps.yml

```diff
@@ -49,6 +49,28 @@
     state: latest
   when: ansible_distribution in ["Fedora", "RedHat", "CentOS"]
 
+- name: Install dependencies (Oracle Linux)
```

Comment:
Same deps as for Centos/RedHat, but I wanted to keep them separated for decoupling.

If you think we can merge them, I'm OK with that too